composer.lock

```diff
@@ -8,16 +8,16 @@
     "packages": [
```

Comment:
Was the `composer.lock` file intended to be in this PR?

Reply:
Sorry, no. I will split this part out and create a new PR out of it.